driver/src/test/java/org/neo4j/driver/integration/RoutingDriverBoltKitIT.java

```diff
@@ -613,6 +614,41 @@ void shouldHandleLeaderSwitchAndRetryWhenWritingInTxFunctionAsync() throws IOExc
         assertThat( writeServer.exitStatus(), equalTo( 0 ) );
     }
 
+    @Test
+    @Disabled("Disabled due the issues with the PR pipeline")
+    void shouldHandleLeaderSwitchAndRetryWhenWritingInTxFunctionAsyncV43() throws IOException, InterruptedException
```

Comment:
do we have the testkit test already? if so maybe we can remove this, otherwise we can leave it as a cleanup task for later

Reply:
It's already in testkit, so i will remove. 😢